### DISCUSSION TITLE
VSCode devcontainer interesting

### DISCUSSION THREAD

**Discussion by elmcrest** [fomantic/Fomantic-UI#1997] [General]
Hey everybody,

probably many of you already know Visual Studio Codes feature to have  a "devcontainer". This is Microsofts shot at supporting "docker based" development - you can just put some config files into your repository and run "reopen in Container" inside of VSCode to have a fully installed development environment, with eslint etc. setup.

So, is this something interesting for fomantic-ui, shold I give it a try and create a PR?

Let me know what you guys think of it...

**Comment by lubber-de** [comment]
🤔 As fomantics usage is meant as some library of a whole project, i can hardly imagine how running fomantic alone in a container environment gives any value. But maybe i am just not getting it at the moment 😉 
However, everything that helps  improving fomantic is welcome, so if you feel this will provide useful value to the project, feel free to provide a PR and related testcases/docs

**Comment by elmcrest** [comment]
The Idea is to get two things:
- a "controlled" environment, easy to start, nearly zero setup, to have something to compare your project against your "actual" project
- a dedicated "compile to css" environment when you just want to serve the resulting css in a different project without a full JS setup

But, actually I'm probably turning to tailwindcss now, but If it's still interesting I could do a PR.

**Comment by lubber-de** [comment]
Do you mean the gitpod approach like here https://github.com/Semantic-Org/Semantic-UI/pull/6941?

**Comment by elmcrest** [comment]
Yeah, seems like that‘s comparable. Didn‘t know gitpod.io yet and I wonder why I‘d prefer a vscode fork and dedicated pricing option over just vscode + docker...
